docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,102 @@ keywords:
   - temporal cloud nexus
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
 :::tip SUPPORT, STABILITY, and DEPENDENCY INFO
 
-Temporal Nexus is available in [Public Preview](https://docs.temporal.io/evaluate/development-production-features/release-stages#public-preview) for both self-hosted deployments and Temporal Cloud with the [Temporal Go SDK](https://docs.temporal.io/develop/go).
+**Temporal Nexus** is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/enable-nexus).
 
 :::
 
-Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries.*
-It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
+## Connect Temporal Applications
+
+Nexus allows you to reliably connect Temporal Applications.
+Designed with Durable Execution in mind, Nexus enables each team to have their own Namespace for improved modularity, security, debugging, and fault isolation.
+
+Nexus Services can be used across and within Namespaces.
+Nexus Operations can span any length of execution, be synchronous or asynchronous, and be implemented with Temporal primitives, like Workflows, or execute arbitrary code.
+
+Unlike other forms of inter-service communication, Nexus combines a familiar programming model with the resiliency of the Temporal Platform and its queue-based Worker architecture.
+The result is an integrated Temporal experience with improved security, observability, and reliability.
+Nexus supports cross-team, cross-domain, cross-namespace, and multi-region use cases.
+
+## Why use Temporal Nexus?
+
+### Integrated developer experience
+
```

Comment:
I'd add a sentence or two here about Nexus and how it helps developer experience before diving into the bullet points. So, basically, an introduction.

Reply:
fixed.